plugins/formal-verification/formver.core/src/org/jetbrains/kotlin/formver/conversion/ContractDescriptionConversionVisitor.kt

```diff
@@ -130,30 +133,37 @@ class ContractDescriptionConversionVisitor(
     ): ExpEmbedding {
         val param = callsEffect.valueParameterReference.accept(this, data)
         val callsFieldAccess = FieldAccess(param, SpecialFields.FunctionObjectCallCounterField)
+        val targetLambdaByCallsEffect = data.functionContractOwner.valueParameterSymbols[callsEffect.valueParameterReference.parameterIndex]
```

Comment:
I would recommend having one way of looking up parameters rather than two separate ones.  In particular, please check that what you implement also works with receivers.

Reply:
Could you do an example with receivers?

Reply:
Do we have a specific test case making use of the receivers you are mentioning?

Reply:
For example: ```kotlin
typealias F = () -> Unit
@OptIn(ExperimentalContracts::class)
fun F.test() {
    contract {
        callsInPlace(this@test)
    }
}
```